person.rb

```diff
@@ -6,10 +6,11 @@ def initialize(age, name = 'Unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rental = []
   end
 
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :id, :rental
```

Comment:
- Can you move this block of  variable declaration above `def initialize()` line? Because you are supposed to declare variables before initialization. Ruby best practices.